src/TrajectoryPlanner/src/TrajectoryGenerator.cpp

```diff
@@ -520,6 +558,11 @@ bool TrajectoryGenerator::getDCMVelocityTrajectory(std::vector<iDynTree::Vector2
     return true;
 }
 
+bool TrajectoryGenerator::getDCMSubTrajectories(std::vector<std::shared_ptr<GeneralSupportTrajectory>> & dcmSubTrajectories)
+{
+    dcmSubTrajectories= m_dcmGenerator->getDCMSubTrajectories();
```

Comment:
The same here, it is better to check if ``isTrajectoyComputed`` return true.

Reply:
Done!